.github/workflows/schematic-api-ci.yml

```diff
@@ -119,6 +119,18 @@ jobs:
               yaml.dump(secrets, file)
         shell: python
 
+      - name: create google credentials
```

Comment:
- "Set up" instead of "Create" would be more accurate
- Capitalize first letter of the step name

Reply:
Does the scope of the credentials adhere to the principle of least privilege? I.e. does the credentials only give access to the Google resources used by this workflow?

Reply:
@tschaffter good question!
@linglp These are google crednetials I created the instructions in the schematic readme.  Are these appropriate for this purpose?